### DISCUSSION TITLE
v2/zh/config/frontmatter/info

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#379] [Announcements]
# v2/zh/config/frontmatter/info

你可以在页面的 frontmatter 配置以下选项设置页面相关信息。 title 类型: string; 必填: 否; 当前页面内容标题，默认为 Markdown 文件中的第一个 h1 标签内容。 shortTitle Type: string; Required: No; 当前页面的短标题，会在导航栏、侧边栏和路径导航中作为首选。 descript...

https://vuepress-theme-hope.github.io/v2/zh/config/frontmatter/info.html

**Comment by pinsily** [comment]
date 时间格式现在是 "YYYY-MM-DD hh:mm:ss" 
版本 2.0.0-beta.130

**Reply by Mister-Hope** [reply]
Thanks for pointing this out.

**Comment by LucaXiang** [comment]
老哥,请问这个banner 怎么用, 添加了没反应

**Reply by Mister-Hope** [reply]
目前仅用于 feed 和 seo 插件
